src/web/builds.rs

```diff
@@ -276,6 +362,119 @@ mod tests {
         });
     }
 
+    #[test]
+    fn build_trigger_rebuild_missing_config() {
+        wrapper(|env| {
+            env.fake_release().name("foo").version("0.1.0").create()?;
+
+            {
+                let response = env.frontend().get("/crate/regex/1.3.1/rebuild").send()?;
+                // Needs POST
+                assert_eq!(response.status(), StatusCode::METHOD_NOT_ALLOWED);
+            }
+
+            {
+                let response = env.frontend().post("/crate/regex/1.3.1/rebuild").send()?;
+                assert_eq!(response.status(), StatusCode::INTERNAL_SERVER_ERROR);
+                let text = response.text()?;
```

Comment:
```suggestion
                let json: serde_json::Value = response.json()?;
```

reqwest has a `.json` method that you can directly use.